eng/pipelines/templates/steps/build-test.yml

```diff
@@ -68,6 +68,13 @@ steps:
         PROXY_CERT: $(Build.SourcesDirectory)/eng/common/testproxy/dotnet-devcert.crt
         ${{ insert }}: ${{ parameters.EnvVars }}
 
+    - ${{ if eq(variables['System.TeamProject'], 'internal')}}:
```

Comment:
I prefer to avoid template expression conditionals like this when possible. I think you can achieve the same effect by passing in a `TestSamples` parameter through from `live.tests.yml` [here](https://github.com/Azure/azure-sdk-for-go/blob/main/eng/pipelines/templates/jobs/live.tests.yml#L87).